target/linux/ath79/dts/qca9563_netgear_wndr.dtsi

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/ignore-/ SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
I couldn't make the firmware unless I remove "ignore-" here.

Reply:
wonder how this made it in. Removed.